### PR TITLE
feat: add pop-up message of dependency installation

### PR DESCRIPTION
**Pull request by richard6094**
feat: add pop-up message of dependency installation

### CODE REVIEW

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -2999,13 +3005,32 @@ export function validateOfficeAddInManifest(args?: any[]): Promise<Result<null,
...
+  const result = await vscode.window.showInformationMessage(notificationMessage, ...buttonOptions);
+
+  if (result === "Yes") {
+    // Handle Yes button click
+    await autoInstallDependencyHandler();
+    await globalStateUpdate(GlobalKey.AutoInstallDependency, false);
+  } else if (result === "No") {
+    // Handle No button click
+    vscode.window.showInformationMessage(
+      "Installation of dependencies is cancelled. But you can install dependencies manually by clicking on the 'Development - Check and install dependencies' button on the left side."
```
Comment (BuWH) [inline packages/vscode-extension/src/handlers.ts:3026]: Localization

File: packages/vscode-extension/src/handlers.ts [hunk 2]
```diff
@@ -2999,13 +3005,32 @@ export function validateOfficeAddInManifest(args?: any[]): Promise<Result<null,
...
-  const terminal = OfficeDevTerminal.getInstance();
-  terminal.show();
-  terminal.sendText(triggerInstall);
+export async function installOfficeAddInDependencies(args?: any[]): Promise<Result<null, FxError>> {
+  popupOfficeAddInDependenciesMessage();
   return Promise.resolve(ok(null));
 }
 
+async function popupOfficeAddInDependenciesMessage() {
+  const buttonOptions = ["Yes", "No"];
```
Comment (ZYUN-MSFT) [inline packages/vscode-extension/src/handlers.ts:3014]: i18n too

File: packages/vscode-extension/src/handlers.ts [hunk 3]
```diff
@@ -2999,13 +3005,32 @@ export function validateOfficeAddInManifest(args?: any[]): Promise<Result<null,
...
+  const notificationMessage = "Install dependencies for Office Add-in?";
```
Comment (ZYUN-MSFT) [inline packages/vscode-extension/src/handlers.ts:3015]: i18n too

File: packages/vscode-extension/src/handlers.ts (hunk 1 above)
Comment (richard6094) [reply packages/vscode-extension/src/handlers.ts]: Added

File: packages/vscode-extension/src/handlers.ts (hunk 3 above)
Comment (richard6094) [reply packages/vscode-extension/src/handlers.ts]: Added

File: packages/vscode-extension/src/globalVariables.ts [hunk 4]
```diff
@@ -60,6 +60,16 @@ export function checkIsSPFx(directory: string): boolean {
   return false;
 }
 
+export function checkOfficeAddInInstalled(directory: string): boolean {
+  const files = fs.readdirSync(directory);
```
[the remark this answers is not in the document]
Comment (richard6094) [reply packages/vscode-extension/src/globalVariables.ts]: Rewrited

**Pull request merged**